pkg/ccl/backupccl/backup_planning.go

```diff
@@ -606,6 +607,19 @@ func backupPlanHook(
 		}
 	}
 
+	var coordinatorLocality roachpb.Locality
+	if backupStmt.Options.CoordinatorLocality != nil {
+		s, err := exprEval.String(ctx, backupStmt.Options.CoordinatorLocality)
+		if err != nil {
+			return nil, nil, nil, false, err
+		}
+		if s != "" {
+			if err := coordinatorLocality.Set(s); err != nil {
+				return nil, nil, nil, false, err
+			}
+		}
+	}
```

Comment:
We'll need to add evaluation of this option to the planning of `ALTER BACKUP SCHEDULE` and `CREATE SCHEDULE ... FOR BACKUP`

Reply:
yeah, I was going to integrate into schedules when I get the proc placement options in as well.

Reply:
Ah, that works for me.